### PR TITLE
Writing review & annotations

### PR DESCRIPTION
**Pull request by frankharkins**
## Changes

Made some suggestions to the phrasing / spelling, added annotations.

### CODE REVIEW

File: notebooks/basics/multiple-systems.ipynb [hunk 1]
```diff
@@ -38,14 +45,25 @@
...
    "id": "847409ef",
-   "metadata": {},
+   "metadata": {
+    "gloss": {
+     "cartesian-product": {
+      "text": "This is a way of combining sets, named after <a href='https://en.wikipedia.org/wiki/Ren%C3%A9_Descartes'>René Descartes</a>.<br> <a href='https://en.wikipedia.org/wiki/Cartesian_product'>Read more</a>.",
+      "title": "Cartesian product"
+     },
+     "string": {
+      "text": "A string is a sequence of symbols. You'll also come across strings in Python, where they are more specifically a sequence of characters.",
```
Comment (jhwatrous) [inline notebooks/basics/multiple-systems.ipynb]: I don't quite understand this. Isn't "character" just another name for "symbol," and if so, why is character more specific?

File: notebooks/basics/multiple-systems.ipynb [hunk 2]
```diff
@@ -317,8 +335,8 @@
     "\n",
     "In this situation it is said that $\\vert \\pi \\rangle$ is a *product state* or *product vector*.\n",
     "\n",
-    "It is common when using the Dirac notation that the tensor product symbol $\\otimes$ is omitted when taking the tensor product of vectors written as kets.\n",
-    "For example, we often write $\\vert \\phi \\rangle \\vert \\psi \\rangle$ rather than \n",
+    "We often omit the '$\\otimes$' when taking the tensor product of kets,\n",
+    "e.g. writing $\\vert \\phi \\rangle \\vert \\psi \\rangle$ rather than \n",
```
Comment (jhwatrous) [inline notebooks/basics/multiple-systems.ipynb]: There should be a comma after e.g.

File: notebooks/basics/multiple-systems.ipynb [hunk 3]
```diff
@@ -1224,7 +1236,14 @@
   {
    "cell_type": "markdown",
    "id": "c30c7bc4",
-   "metadata": {},
+   "metadata": {
+    "gloss": {
+     "john-bell": {
+      "text": "John Bell was a physicist known for his work on quantum mechanics. We'll cover a famous theorem of his later in this course.",
```
Comment (jhwatrous) [inline notebooks/basics/multiple-systems.ipynb]: I don't plan to refer to "Bell's Theorem" specifically in this way. We could say "We will discuss one of his most famous discoveries in Lesson 4."

File: notebooks/basics/multiple-systems.ipynb (hunk 1 above)
Comment (frankharkins) [reply notebooks/basics/multiple-systems.ipynb]: Good point, I think in Python a "character" is a specific object, but it's probably clearer if I remove this sentence. I think I just wanted to make the tooltip longer.

**Review by jhwatrous — APPROVED** [review]

**Pull request merged**